keylime/mba/mba.py

```diff
@@ -0,0 +1,100 @@
+import json
+from typing import Dict, Optional, Set
+
+from keylime import config
+from keylime.failure import Failure
+from keylime.mba.elchecking import mbpolicy, policies
+
+# ###########
+# when something goes wrong with MBA policies
+# ###########
+
+
+class PolicyException(Exception):
+    """
+    This exception is raised if policies cannot be loaded.
+    TODO final format including a reason code.
```

Comment:
This will come later in this PR, right? Would make sense to move into Draft until all the code is there?

Reply:
@aplanas My main argument in favor of merging this particular PR, the first of "many" (at least 3) is that we have a good precedent in the project of effecting these significant changes in an incremental manner. It was so when we first introduced "measured boot" (3 PRs) and "durable attestation" (4 PRs, with probably more needed).

Reply:
Maybe we can change it to a ValueError for now and introduce the specific exception when needed.

Reply:
done (ValueError)